src/utils/itemValidator.js

```diff
@@ -0,0 +1,27 @@
+export const isInputEmpty = (userInput) => {
+	const trimmedUserInput = userInput.trim();
+	return trimmedUserInput ? false : true;
+};
+
+export const isItemDuplicate = (newItem, itemsList) => {
+	const removePunctuation = (inputItem) => {
+		const punctuation = /[.,?!]/g;
```

Comment:
Nitpick: for the punctuation regex I think it is worth it to consider adding a few more symbols/punctuation. ex: @,#,$,%,&, etc. As this could help further restrict(broaden the scope of what is checked for) what is able to be entered. I was able to add an item that was "milk&" for example.

Reply:
I added some more in!